packages/cf.js/src/app-factory.ts

```diff
@@ -1,11 +1,50 @@
-import { Address, AppABIEncodings } from "@counterfactual/common-types";
+import {
+  Address,
+  AppABIEncodings,
+  AppInstanceID,
+  AppState,
+  BlockchainAsset,
+  Node
+} from "@counterfactual/common-types";
+import { ethers } from "ethers";
```

Comment:
@mvanderh We could also use:

```ts
import { BigNumber, BigNumberish } from "ethers/utils";
```

like you did on the AppInstance file.

Reply:
Ah yea, this is true.